app/routes/planet_routes.py

```diff
@@ -0,0 +1,73 @@
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, make_response, request
+from .helpers import validate_planet
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+# Get all planets
+@planets_bp.route("", methods=["GET"])
+def get_all_planets():
+    name_query = request.args.get("name")
+    id_query = request.args.get("id")
+    if name_query:
+        planets = Planet.query.filter_by(name = name_query)
+    elif id_query:
+        planets = Planet.query.filter_by(id = id_query)
+    else:
+        planets = Planet.query.all() 
+
+    planets_response = []
+    # planets = Planet.query.all()
+    for planet in planets:
+        planets_response.append(planet.to_json())
+    
+    return jsonify(planets_response),200
+
+# Get one planet
+@planets_bp.route("/<id>", methods=["GET"])
+def get_one_planet(id):
+    planet = validate_planet(id)
+
+    return jsonify(planet.to_json()),200
+
+# Create planets
+@planets_bp.route("", methods=["POST"])
+def create_planets():
+    request_body = request.get_json()
+    new_planet = Planet.create(request_body)
+    db.session.add(new_planet)
+    db.session.commit()
+    return make_response(jsonify(f"Planet {new_planet.name} successfully created"), 201)
+    # return make_response(f"Planet {new_planet.name} successfully created", 201)
+# Update one planet
+@planets_bp.route("/<id>", methods=["PUT"])
+def update_one_planet(id):
+    planet = validate_planet(id)
+    request_body = request.get_json()
+    planet.update(request_body)
+    db.session.commit()
+    
+    return make_response(f"Planet {id} successfully updated", 200)
+
+# Delete one planet
+@planets_bp.route("/<id>", methods=["DELETE"])
+def delete_one_planet(id):
+    planet = validate_planet(id)
+    db.session.delete(planet)
+    db.session.commit()
+    return make_response(f"Planet {id} successfully deleted", 200)
```

Comment:
When we pass a string to `make_response`, Flask will assume that we want to return an HTML rather than JSON data. `make_response` is useful when we want to attach headers to a response but in this case, we're simply returning JSON, so `jsonify()` is sufficient enough. 

```python
return jsonify(f"Planet {id} successfully deleted"), 200
```